main/src/ca/uwaterloo/flix/language/ast/WeededAst.scala

```diff
@@ -299,11 +299,17 @@ object WeededAst {
 
   object ExtPattern {
 
-    case class Wild(loc: SourceLocation) extends ExtPattern
+    sealed trait VarOrWild {
```

Comment:
This does not extend ExtPattern. The structure here is a bit confusing to me.

I suppose there is only one ExtPattern, namely Tag. which then has VarOrWild?

What about rearranging:

```
case class ExtTag(label: Name.Label, pats: List[ExtTagPat], loc: SourceLocation) extends ExtPattern

sealed trait ExtTagPat  // could also be called ExtTagFragment

object ExtTagPat {
 // Wild, Var, and Error
}
```

Reply:
Having ExtPattern as a trait makes little sense when it only has one implementation.